source/gui/settingsDialogs.py

```diff
@@ -2992,6 +2992,33 @@ def makeSettings(self, settingsSizer: wx.BoxSizer) -> None:
 
 		self._appendSoundSplitModesList(sHelper)
 
+		# Translators: This is the label for a slider control in the
+		# Audio settings panel.
+		label = _("Volume of other applications")
+		self.appSoundVolSlider: nvdaControls.EnhancedInputSlider = sHelper.addLabeledControl(
+			label, nvdaControls.EnhancedInputSlider,
+			minValue=0,
+			maxValue=100,
+		)
+		self.bindHelpEvent("OtherAppVolume", self.appSoundVolSlider)
+		volume = config.conf["audio"]["applicationsSoundVolume"]
+		if 0 <= volume <= 100:
+			self.appSoundVolSlider.SetValue(volume)
+		else:
+			log.error("Invalid volume level: {}", volume)
+			defaultVolume = config.conf.getConfigValidation(["audio", "applicationsSoundVolume"]).default
+			self.appSoundVolSlider.SetValue(defaultVolume)
+
+		# Translators: This is a label for the applications volume adjuster combo box in settings.
+		label = _("&Application volume adjuster mode")
+		self.appVolAdjusterCombo: nvdaControls.FeatureFlagCombo = sHelper.addLabeledControl(
```

Comment:
The UX in the Audio panel with this combo is a bit strange. When you modify this combo box, this enables or disables the slider which is above it.

If there was no specific reason to position them as you did in the panel, could you switch them, i.e. put the combo box of the volume adjuster above the other apps volume slider?